docs/platform_concepts/inbox/inbox-settings/chats-configuration/chats-configuration.md

```diff
@@ -0,0 +1,287 @@
+---
+sidebar_label : WhatsApp Campaign
```

Comment:
Give appropriate sidebar label, this doesn't seem like WhatsApp Campaign doc. 
Also add the filepath to the sidebars.js. It's not showing up.
Remove the content which is already published somewhere else. For i.e. we already have separate page for Tags, so you can remove that content from here.

Reply:
Done.